spec.html

```diff
@@ -14441,13 +14516,32 @@ <h1>Runtime Semantics: DestructuringAssignmentEvaluation</h1>
             1. If IsAnonymousFunctionDefinition(|Initializer|) is *true*, then
               1. Let _hasNameProperty_ be ? HasOwnProperty(_v_, `"name"`).
               1. If _hasNameProperty_ is *false*, perform SetFunctionName(_v_, _P_).
-          1. Return ? PutValue(_lref_, _v_).
+          1. Perform ? PutValue(_lref_, _v_).
+          1. Return a new List containing _P_.
         </emu-alg>
+
         <emu-grammar>AssignmentProperty : PropertyName `:` AssignmentElement</emu-grammar>
         <emu-alg>
           1. Let _name_ be the result of evaluating |PropertyName|.
           1. ReturnIfAbrupt(_name_).
-          1. Return the result of performing KeyedDestructuringAssignmentEvaluation of |AssignmentElement| with _value_ and _name_ as the arguments.
+          1. Let _status_ be the result of performing KeyedDestructuringAssignmentEvaluation of |AssignmentElement| with _value_ and _name_ as the arguments.
+          1. ReturnIfAbrupt(_status_).
+          1. Return a new List containing _name_.
+        </emu-alg>
+      </emu-clause>
+
+      <emu-clause id="Rest-RuntimeSemantics-RestDestructuringAssignmentEvaluation">
+        <h1>Runtime Semantics: RestDestructuringAssignmentEvaluation</h1>
+        <p>With parameters _value_ and _excludedNames_.</p>
+
+        <emu-grammar>AssignmentRestProperty[Yield, Await] : `...` DestructuringAssignmentTarget</emu-grammar>
+        <emu-alg>
+          1. Let _lref_ be the result of evaluating |DestructuringAssignmentTarget|.
+          1. ReturnIfAbrupt(_lref_).
+          1. Let _restObj_ be ObjectCreate(%ObjectPrototype%).
+          1. Let _assignStatus_ be CopyDataProperties(_restObj_, _value_, _excludedNames_).
+          1. ReturnIfAbrupt(_assignStatus_).
```

Comment:
This line should be removed, and the preceding `CopyDataProperties ` be prefixed with a `?`